### PR TITLE
Scripts/Naxxramas: Finally fix Frogger event

### LINKED ISSUE #16488: [3.3.5] DB/Creature: Frogger pathing
3.3.5 d1a185e3d5af

Frogger spawn in Naxxramas is incorrect. It should go perpendicular to the road across the entire width, instead of diagonal only across the right half.

### PR DESCRIPTION
**Pull request by Treeston**
What the title says.

Thanks Killyana for the sniffed coords. Closes #16488.

### CODE REVIEW

File: src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp [hunk 1]
```diff
@@ -477,6 +480,17 @@ class instance_naxxramas : public InstanceMapScript
                                 kelthuzad->AI()->Talk(CurrentWingTaunt);
                             ++CurrentWingTaunt;
                             break;
+                        case EVENT_SUMMON_FROGGER_WAVE:
+                        {
+                            static const uint32 FROGGER_ENTRY = 16027;
```
Comment (P-Kito) [inline src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp]: Why is this here

Comment (Treeston) [inline src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp]: Why not. I considered putting it just outside the method, but it looks weird there, too.

**Comment by tkrokli** [comment]
You may want to do a new push (refresh) to your own repo, now that Travis is working again. :-)

**Comment by mik1893** [comment]
is there a reason why we are not merging this?

**Pull request merged**